ingest_api/infrastructure/config.py

```diff
@@ -87,3 +95,17 @@ def env(self) -> aws_cdk.Environment:
             account=self.aws_account,
             region=self.aws_region,
         )
+
+    @property
```

Comment:
I think I would prefer to keep it in the ingest_api for now since, in the future, we want to block access via apigw anyway, right? 🤔 I'm open to moving it though!

Reply:
Let's keep it here then 👍